hikari/impl/rest.py

```diff
@@ -759,6 +765,23 @@ async def _request(
                     real_url = str(response.real_url)
                     raise errors.HTTPError(f"Expected JSON [{response.content_type=}, {real_url=}]")
 
+                # Handling 5xx errors
+                if 500 <= response.status < 600 and retries_done <= self._max_retries and self._max_retries != 0:
+                    if backoff is None:
+                        backoff = rate_limits.ExponentialBackOff()
+
+                    sleep_time = next(backoff)
+                    _LOGGER.warning(
+                        "Received status %s on request, backing off for %ss and retrying. Retries remaining: %s",
+                        response.status,
+                        round(sleep_time, 1),
```

Comment:
No need to round, being verbose is better here, as this is meant for debug purposes

```suggestion
                    _LOGGER.warning(
                        "Received status %s on request, backing off for %.2fs and retrying. Retries remaining: %s",
                        response.status,
                        sleep_time,
```